edge/pkg/metamanager/dao/meta.go

```diff
@@ -47,6 +48,17 @@ func DeleteMetaByKey(key string) error {
 	return err
 }
 
+// DeleteMetaByKeyAndPodUID delete meta by key and podUID
+func DeleteMetaByKeyAndPodUID(key, podUID string) (int64, error) {
+	sqlStr := fmt.Sprintf("DELETE FROM meta WHERE key = '%s' and value LIKE '%%%s%%'", key, podUID)
+	res, err := dbm.DBAccess.Raw(sqlStr).Exec()
+	if err != nil {
+		klog.Errorf("delete pod by key %s and podUID failed, err: %v", key, podUID)
```

Comment:
seem that log is wrong, maybe like this?
`
klog.Errorf("delete pod by key %s and podUID %s failed, err: %v", key, podUID, err)
`

Reply:
> seem that log is wrong, maybe like this? `klog.Errorf("delete pod by key %s and podUID %s failed, err: %v", key, podUID, err)`

yes， I have fixed it.